src/Validator/DocumentValidator.php

```diff
@@ -260,14 +257,23 @@ public static function validateSDL(
         ?Schema $schemaToExtend = null,
         ?array $rules = null
     ): array {
-        $usedRules = $rules ?? self::sdlRules();
+        $rules ??= self::sdlRules();
+
+        if (0 === count($rules)) {
```

Comment:
btw why are we using yoda conditions? 🤔

Reply:
Because of `php-cs-fixer`. I don't mind them - do you have a strong opinion about it?

Reply:
Hm, it makes reading code hard because that's not how people speak. 

I have never seen anyone to use assign instead comparison in a condition. Also we have phpstan and tests to prevent that. So there's no use for yoda.